api/src/firstrun.py

```diff
@@ -333,6 +333,31 @@ def create_shop_accounts_cost_centers():
     db_session.commit()
 
 
+def create_shop_gift_cards():
+    banner(YELLOW, "Creating Fake Gift Cards and Gift Card & Product Mappings")
+
+    gift_card = get_or_create(
+        GiftCard,
+        amount=299.00,
+        validation_code=12989519,
+        email="test@fake.com",
+        status="activated"
+    )
+
+    # Get existing product with ID: 64 (Makerspace access starter pack)
+    product = get_or_create(
+        Product,
+        id=64,
+    )
```

Comment:
An improvement that can be done here is to fetch the makerspace access starter pack more directly instead of using the id. The id might change if some other product gets added to first run

```suggestion
    product = get_or_create(
        Product,
        name="Makerspace access starter pack",
    )
```